dataedit/views.py

```diff
@@ -758,6 +759,37 @@ def view_delete(request, schema, table):
     return redirect("/dataedit/view/" + schema + "/" + table)
 
 
+def create_graph(request, schema, table):
+
+    if request.method == 'POST':
+        # save an instance of View, look at GraphViewForm fields in forms.py for information to the
+        # options
+        opt = dict(x=request.POST.get('column_x'), y=request.POST.get('column_y'))
+        gview = View.objects.create(
+            name=request.POST.get('name'),
+            table=table,
+            schema=schema,
+            type='graph',
+            options=opt,
+            is_default=request.POST.get('is_default')
+        )
+        gview.save()
+
+        return redirect(
+            "/dataedit/view/{schema}/{table}".format(schema=schema, table=table)
+        )
+    else:
+        # TODO get the columns from the schema and the table
+        columns = [
```

Comment:
@MGlauer, @gnn - here how could I get the columns of the table with the table and schema's name. Is there a function of the api for that?

Reply:
You could use describe_columns from actions.py like this:
describe_columns(schema, table).keys()

Reply:
Thanks :)